convert/src/vertex_to_bd/badgerdoc_format/annotation_converter.py

```diff
@@ -0,0 +1,118 @@
+from typing import Any, List, Optional, Tuple
+
+from ..models import LabelStudioModel, ModelItem, ResultItem
+from ..models.bd_annotation_model import BadgerdocAnnotation, Obj, Page, Size
+from ..models.bd_tokens_model import Page as BadgerdocTokensPage
+
+LABELS = "labels"
+RELATION = "relation"
+
+
+class AnnotationConverter:
+    def convert(
+        self,
+        annotations: LabelStudioModel,
+        badgerdoc_tokens: BadgerdocTokensPage,
+    ):
+        model_items: List[ModelItem] = annotations.__root__
+        badgerdoc_annotations = BadgerdocAnnotation(
+            pages=[Page(page_num=1, size=Size(width=100, height=100))]
```

Comment:
May you clarify these exact values here?

Reply:
Hi, annotation converter part is still in progress. Actually, even requirements are not clarified.

Reply:
https://github.com/epam/badgerdoc/issues/170